dockerfiles/pytorch-dev/install_rocm_from_tarballs.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -euo pipefail
+
+# Path to the directory containing ROCm tarballs
+# TODO find the correct path for the artifact tarballs
+TARBALL_DIR="${1:-/rocm-tarballs}"
+INSTALL_PREFIX="/opt/rocm"
+
+echo "Installing ROCm from tarballs in $TARBALL_DIR"
+
+# 1. Create installation directory
+sudo mkdir -p "$INSTALL_PREFIX"
+
+# 2. Extract all tarballs to /opt/rocm
+for tarball in "$TARBALL_DIR"/rocm-*.tar.gz; do
+  echo "Extracting: $tarball"
+  sudo tar -xvzf "$tarball" -C "$INSTALL_PREFIX"
+done
+
+# 3. Set up environment variables
+ROCM_ENV_FILE="/etc/profile.d/rocm.sh"
+echo "Writing environment variables to $ROCM_ENV_FILE"
+
+sudo tee "$ROCM_ENV_FILE" > /dev/null <<EOF
+export PATH=$INSTALL_PREFIX/bin:\$PATH
+export LD_LIBRARY_PATH=$INSTALL_PREFIX/lib:$INSTALL_PREFIX/lib64:\$LD_LIBRARY_PATH
```

Comment:
@erman-gurses , you don't need to set `LD_LIBRARY_PATH` if the same directory is in a `ld.so.conf.d` snippet, the dynamic linker is already instructed to look in that directory by the latter.

Reply:
@scottt, Thanks for the info in advance